node/src/actors/chain_manager/handlers.rs

```diff
@@ -834,6 +834,35 @@ impl PeersBeacons {
             .collect()
     }
 
+    /// Remove beacons that are old or has a different hash if is the same super epoch
+    pub fn remove_no_consensus_beacons(&mut self, superbeacon: CheckpointBeacon) {
+        let mut valid_peers = vec![];
+
+        for (addr, last_beacon) in &self.pb {
+            if let Some(last_beacon) = last_beacon {
+                // Remove beacons that would have an old superblock checkpoint
+                if last_beacon.highest_superblock_checkpoint.checkpoint < superbeacon.checkpoint
+                    // Remove beacons that are different from the consensus superblock
+                    || (last_beacon.highest_superblock_checkpoint.checkpoint
+                        == superbeacon.checkpoint
+                        && last_beacon.highest_superblock_checkpoint != superbeacon)
+                {
+                    // Peers out of consensus there will be handle as no beacon peers
```

Comment:
```suggestion
                    // Peers out of consensus will be treated the same peers from which we receive no beacon
```

Reply:
shouldn't it be "will be treated as..."?

Reply:
Oops, yes, I meant "treated the same as peers...."

Reply:
Ping @lrubiorod